### DISCUSSION TITLE
How do I confirm?

### DISCUSSION THREAD

**Discussion by MAX1MDEV** [dyc3/steamguard-cli#422] [Q&A]
SDA had a window to confirm actions. I need to create a web key and I don't see anything indicating that.

**Comment by MAX1MDEV** [comment]
![image](https://github.com/user-attachments/assets/aa081ebd-865b-48bf-af8b-ad1f5f6c781a)
![image](https://github.com/user-attachments/assets/7333dd29-1b15-4275-931b-9c1eb40cd137)
doesnt work

**Reply by dyc3** [reply]
I can't read Russian. You'll need to elaborate a little more if you want me to understand what you are trying to do.

Steamguard cli uses the exact same apis that the mobile app uses for confirmations. If it's not showing a confirmation to you, and steam is asking for a mobile confirmation theres not a lot I can do about it.
